backend/tests/owasp/models/post_test.py

```
@@ -0,0 +1,80 @@
+    def test_bulk_save(self):
+        """Test the bulk_save method."""
+        mock_posts = [Mock(id=None), Mock(id=1)]
+        with patch("apps.owasp.models.post.BulkSaveModel.bulk_save") as mock_bulk_save:
+            Post.bulk_save(mock_posts, fields=["title"])
+            mock_bulk_save.assert_called_once_with(Post, mock_posts, fields=["title"])
+
+    @pytest.mark.parametrize(
+        ("data", "expected_fields"),
+        [
+            (
+                {"title": "New Post", "url": "https://example.com", "author_name": "John Doe", "published_at": datetime(2025, 1, 1), "author_image_url": "https://image.com"},
```

Ruff (0.8.2)

Line too long (174 > 99)

(E501)